SportHub/src/modules/AdminArticlesList/components/AdminArticlesList.jsx

```diff
@@ -43,16 +49,26 @@ function AdminArticlesList({}) {
         }
     }
 
+    useEffect(() => {
+        if (typeof (setButtons) == "function") {
+            setButtons([
+                { text: "Create article", function: onCreateArticleHandler, isOutlined: false }
+            ])
+        }
+        setCurrentPageNumber(1)
+    }, [])
+    useEffect(() => {
+        setArticles([])
+        setCurrentPageNumber(1)
```

Comment:
1 can be moved to constant like firstPageNumber -> this can be used everywhere where first page needed